src/action_jailbuild.rs

```diff
@@ -0,0 +1,23 @@
+use crate::rua_dirs::TARGET_SUBDIR;
+use crate::{tar_check, wrapped};
+use directories::ProjectDirs;
+use std::fs;
+use std::path::PathBuf;
+
+pub fn action_jailbuild(offline: bool, target: PathBuf, dirs: &ProjectDirs) {
+	let target_str = target
```

Comment:
this code is copy-paste from main.rs, no changes.

Reply:
Then why not have it in its own commit?

Reply:
I guess I could do that, didn't think of such a desire. Do you review changes per-commit?

If you want, I can split into 2 commits and force-push.

Reply:
I'm not super anal about other people's repos but I do tend to subscribe to the "one working change" per commit ideology.

>Do you review changes per-commit?

Depends, in this case I could see that commit one is just refactoring so there's no need to read over it.

Reply:
Done. I've split it into 2 commits and pushed.

Reply:
OK to merge / did you finish review?

Reply:
It looks fine to me. But I'm just casually reading the code. I have not tried to run it.

Although if you wan't a slight optimisation, sort the search terms by size first.